src/components/floaty-object.js

```diff
@@ -1,14 +1,10 @@
+import { addComponent } from "bitecs";
+import { FloatyObject } from "../bit-components";
+import { FLOATY_OBJECT_FLAGS } from "../systems/floaty-object-system";
 /* global AFRAME */
-const COLLISION_LAYERS = require("../constants").COLLISION_LAYERS;
 
 AFRAME.registerComponent("floaty-object", {
```

Comment:
Note all this functionality moved into the new system. This component now only exists to be used in hub.html, once nothing is using it there anymore the aframe component part can be removed.